requests.lwr

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
`python-requests` is already a recipe, and is in PyBOMBS default recipe location.